tests/rest/server.go

```diff
@@ -210,15 +288,30 @@ func checkOpenAPISpecifications() {
 
 // ServerTests run all tests for basic REST API endpoints
 func ServerTests() {
+	// basic tests for REST API apiPrefix
 	checkRestAPIEntryPoint()
 	checkNonExistentEntryPoint()
 	checkWrongEntryPoint()
 	checkWrongMethodsForEntryPoint()
+
+	// tests for REST API endpoints apiPrefix+"organizations"
 	checkOrganizationsEndpoint()
 	checkOrganizationsEndpointWrongMethods()
+
+	// tests for REST API endpoints apiPrefix+"report/{organization}/{cluster}"
 	checkClustersEndpointForKnownOrganizations()
 	checkClustersEndpointForUnknownOrganizations()
 	checkClustersEndpointForImproperOrganizations()
 	checkClustersEndpointWrongMethods()
+
+	// tests for REST API endpoints apiPrefix+"report/{organization}/{cluster}"
+	checkReportEndpointForKnownOrganizationAndKnownCluster()
+	checkReportEndpointForKnownOrganizationAndUnknownCluster()
+	checkReportEndpointForUnknownOrganizationAndKnownCluster()
+	checkReportEndpointForUnknownOrganizationAndUnknownCluster()
+	checkReportEndpointForImproperOrganization()
+	checkReportEndpointWrongMethods()
+
+	// tests for OpenAPI specification that is accessibla via its endpoint as well
```

Comment:
`// tests for OpenAPI specification that is accessible via its endpoint as well`

Reply:
where's problem?

Reply:
accessibla should be accessible

Reply:
ah, my eyes! ok sorry

Reply:
fixed